src/Bicep.RegistryModuleTool.IntegrationTests/Bicep.RegistryModuleTool.IntegrationTests.csproj

```diff
@@ -0,0 +1,23 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
```

Comment:
>TargetFramework [](http://example.com/codeflow?start=5&length=15)

This one is set at repo-level for all *.csproj, so it can be removed from all the projects.

Reply:
Yup. It seems VS is not smart enough to omit it when creating a new project. Removed it.